docs/subquery_network/node_operators/stake.md

```diff
@@ -0,0 +1,68 @@
+
+:::tip New for Mainnet - Please read
+
+One key difference between Kepler and the mainnet is that now Node Operators must actively manage their staking allocation to each project deployment.
+
+:::
+
+In order to register and become a Node Operator, a minimum amount of SQT is required to stake in the staking contract.
+
+If a Node Operator is caught misbehaving (such as by providing invalid, incomplete, or incorrect data), they are liable to have a portion of their staked SQT (on the particular reward pool) reallocated to the SubQuery Foundation Treasury, diminishing their staked SQT in the network and therefore reducing their potential rewards.
+
+## Managing Staking Allocation
+
+Node Operators can always see their current own stake, total delegation, and any unallocated stake in the Indexer > My Projects menu.
+
+![indexer stake manage screen](/assets/img/network/indexer_stake_manage.png)
+
+At the top of the page shows the current staking amount (and the amount that is estimated for the next Era).
+
+- Own Stake: This is the total current amount of SQT that you as the Node Operator has staked. Remember, you must meet a minimum ratio here to delegated stake - see [how much SQT should you stake](#how-much-sqt-should-you-stake).
+- Total Delegation: This is the total amount of delegated SQT to you as an Node Operator.
+- Unallocated Stake: This is the amount of stake that is not allocated to any project deployment. **You should always try to minimise this to as close to 0**. See how to [allocate stake](#allocating-stake).
+- Over Allocated Stake: **This is really bad** - please immediately see [over allocated stake](#over-allocated-stake).
+
+You can add or remove Own Stake by clicking on the "Stake More" or "Unstake" buttons shown below.
+
+![indexer stake add screen](/assets/img/network/indexer_stake_add.png)
+
+### Allocating Stake
+
+:::warning
+
+When your stake is unallocated, it's not being applied to the stake of any particular project, and is not doing any productive work for that project deployment - in short, it's wasted.
+
+:::
+
+You can add or remove allocation for a particular project deployment by clicking "Add Allocation" or "Remove Allocation" next to each project deployment.
+
+![indexer stake manage screen](/assets/img/network/indexer_stake_manage_add.png)
+
+Remember, both network inflation rewards and query rewards (for Flex plans) are distributed for based on the Cobb-Douglas production ([read more here](./rewards.md#how-are-node-operators-rewarded)). This means that you should allocate a larger proportion of stake to project to which you derive more rewards in these two areas. For example:
+
+- If a project deployment is highly boosted, allocating more stake means you receive a higher proportion of its rewards compared to other project deployments
+- If a project deployment gets a lot of flex plan queries, allocating more stake means you receive a higher proportion of its rewards compared to other project deployments
```

Comment:
flex plan(payg) doesn't use allocated SQT in current version. We do want to switch to it in the future

Reply:
I think leave it like this for now